pbs_gadi_runscript.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+#PBS -P er01
+#PBS -N fastqc-nf
+#PBS -l walltime=01:00:00
+#PBS -l ncpus=1
+#PBS -l mem=40GB
```

Comment:
This is too much RAM for 1 CPU on copyq. Should be 4 GB max. In its current form, will be chared 40/4 = 10 X the SU charge rate for this queue.